src/main/java/services/event_creation/EventInfoFromReader.java

```diff
@@ -49,4 +50,17 @@ public LocalTime getEndTime() {
     public boolean getCompleted() {
         return eventReader.getCompleted();
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
why are adding methods purely for the test suite? or was there some larger intention when including this.
Also if you're going to override equals, might as well make it property-based, allowing for equality across other EventInfo instead of a wrapper based comparison with another concrete EventInfoFromReader. The current method exposes the underlying implementation underneath the abstraction, making it so that client code has to be aware that the two items it's comparing are EventInfoFromReader in order to compare equality (it should be something more abstract like the properties are the same)

Reply:
OK I deleted the equals method